src/flatfile.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <flatfile.h>
+#include <tinyformat.h>
+
+FlatFileSeq::FlatFileSeq(fs::path dir, const char* prefix, size_t chunk_size)
+    : m_dir(std::move(dir)), m_prefix(prefix), m_chunk_size(chunk_size)
```

Comment:
Commit 88cf780

nit, initialize one per line?

Reply:
I don't see this in the style guide?

Reply:
You also don't see to declare class members in multiple lines, but fair enough.